mozilla/fx-private-relay
# Allow manually creating custom aliases

# New feature description

This allows users to create custom aliases themselves, without having to send an email to it.

# Screenshot (if applicable)

How about a video?

https://user-images.githubusercontent.com/4251/150332894-7319960d-621b-412e-88b3-4b6645aa3e49.mp4

# How to test

1. Log in using an account that has Premium _and_ has a custom subdomain.
2. Click "Generate new alias".
3. Verify that a menu pops up, through which you can create your own custom alias.
4. Also make sure that a non-Premium user, or a Premium user without a custom subdomain, just has a regular button that directly creates a random alias.

# Checklist

- [x] l10n dependencies have been merged, if any. - Most of them, but I forgot two unfortunately: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/55
- [ ] All acceptance criteria are met. - It also says it should shop up on top, but alias sorting I think is a separate PR that might conflict with https://github.com/mozilla/fx-private-relay-add-on/pull/242, so I left that for a potential follow-up PR.
